### PR TITLE
Autowiki Test

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request

Adds the autowiki workflow.

Its only for vendors. Dont care to unravel whatever the techweb has going on.

Ports:
- https://github.com/tgstation/tgstation/pull/64417
- https://github.com/tgstation/tgstation/pull/70779
- https://github.com/tgstation/tgstation/pull/71919

## Why It's Good For The Game

The amount of deprecated crap in our wiki is getting old. The amount of hours I've spent removing garbage we havent had for FIVE years. Guh.

May as well give it a try

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl: rkz, mothblocks, Zonespace27, 
server: Adds autowiki workflow and adds it to vendors.
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
Needs TM to show Testing evidence

File: code/controllers/subsystem/shuttle.dm [hunk 1]
```diff
@@ -360,6 +360,8 @@ SUBSYSTEM_DEF(shuttle)
 			hostileEnvironments -= d
 	emergencyNoEscape = hostileEnvironments.len
 
+	if(!emergency)
+		message_admins("[src] has no emergency dock? What fuckery is this?")
```
Comment (EvilDragonfiend) [inline code/controllers/subsystem/shuttle.dm:364]: why

**Comment by EvilDragonfiend** [comment]
DNM until Crossed approves

**Comment by EvilDragonfiend** [comment]
@Crossedfall I think there should be a dummy mediawiki or something to test this system. Some ingame descriptions are outdated, and we wouldn't want to fool wiki users.

**Comment by Tsar-Salat** [comment]
> Some ingame descriptions are outdated

Yeah, im calling the bluff on this. Vending machine wikipage hasnt been updated in 3 years.

image


> I think there should be a dummy mediawiki or something to test this system.

> and we wouldn't want to fool wiki users

Thats... the point of this pr? Vending machines get no traffic. It *is* the test.

I edited the page a month ago.

**Comment by Tsar-Salat** [comment]
It took over 6 months for someone to even notice the vending machine page was changed to not have info.

I do not think the wiki experience will be impacted if it doesnt work right, we can just revert the changes if it doesnt work out.

**Review by PowerfulBacon — APPROVED** [review]
Test this locally to make sure the DM code works fine, then I will merge this into its own branch for testing the wiki stuff before merging into master.

**Pull request merged**